ibm/service/vpc/data_source_ibm_is_volume.go

```diff
@@ -271,6 +303,30 @@ func volumeGet(d *schema.ResourceData, meta interface{}, name string) error {
 		}
 		d.Set(isVolumeStatusReasons, statusReasonsList)
 	}
+	if vol.HealthReasons != nil {
+		healthReasonsList := make([]map[string]interface{}, 0)
+		for _, sr := range vol.HealthReasons {
+			currentSR := map[string]interface{}{}
+			if sr.Code != nil && sr.Message != nil {
+				currentSR[isVolumeHealthReasonsCode] = *sr.Code
+				currentSR[isVolumeHealthReasonsMessage] = *sr.Message
+				if sr.MoreInfo != nil {
+					currentSR[isVolumeHealthReasonsMoreInfo] = *sr.Message
+				}
+				healthReasonsList = append(healthReasonsList, currentSR)
+			}
+		}
+		d.Set(isVolumeHealthReasons, healthReasonsList)
+	}
+	if vol.HealthState != nil {
+		d.Set(isVolumeHealthState, *vol.HealthState)
+	}
+	tags, err := flex.GetTagsUsingCRN(meta, *vol.CRN)
+	if err != nil {
+		log.Printf(
+			"Error on get of resource vpc volume (%s) tags: %s", d.Id(), err)
+	}
+	d.Set(isVolumeTags, tags)
```

Comment:
Why are we setting tags twice 
d.Set(isVolumeTags, vol.UserTags)
d.Set(isVolumeTags, tags)